code-review.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title:  "Code Review"
```

Comment:
Can/must the reviewer update the source code if he finds something that can be easily improved/fixed?
If the reviewer performs some updates, who does review them?

Reply:
"If during the review, the reviewer decides to make any changes to the code, instead of giving a status to the review, he assigns back the review to the branch owner for approval."

I move this from `development` to `code review` since it seems a better location.